frontend/src/Form.tsx

```diff
@@ -36,47 +40,27 @@ type State = Record<string, string[]> & {
   cells: string[]
 }
 
+const specific_key = (s: SpecificOption) => s.column + ',' + s.tumor
+
 function calculate_state0(conf: Conf, key_prefix: string): State {
   const state0 = {} as State
   conf.variant_values.forEach(v => {
     state0[v.column] = v.values
   })
   conf.tumor_specific_values.forEach(v => {
-    state0[v.column + ',' + v.tumor] = v.values
+    state0[specific_key(v)] = v.values
   })
-  state0.cells = []
+  state0.cells = [conf.cells.includes('CD4') ? 'CD4' : conf.cells[0]]
```

Comment:
I think it'd look prettier with having 'CD4' in a `default_value` constant, but it's not important :)